tests/bench_sizeof_coretypes/main.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2014 René Kijewski <rene.kijewski@fu-berlin.de>
+ *               2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Print size of core types
+ *
+ * @author      René Kijewski <rene.kijewski@fu-berlin.de>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "cib.h"
+#include "clist.h"
+#include "panic.h"
+#include "kernel_types.h"
+#include "list.h"
+#include "mbox.h"
+#include "msg.h"
+#include "mutex.h"
+#include "priority_queue.h"
+#include "ringbuffer.h"
+#include "rmutex.h"
+#ifdef MODULE_CORE_THREAD_FLAGS
+#include "thread_flags.h"
+#endif
+#include "thread.h"
+
+
+#define P(NAME) printf("    tcb->%-11s            %3u     %3u\n", #NAME, \
```

Comment:
uncrustify is saying that there should be a space between `#` and `NAME`, which I'm not sure is true. Any thoughts on this? Error in the uncrustify.cfg/limitation to uncrustify?

Reply:
I disagree with uncructify, there should be no space here.

Reply:
Yeah sure. I'll put a comment in Jose's PR linking to this discussion

Reply:
this is a case where uncrustify's `/** INDENT-OFF */` and `/**^INDENT-ON*/` would totally make sense :)

Reply:
I agree with @haukepetersen and I disagree, that `/** INDENT-OFF */` should be used here (side-note: why are doxygen comments required for this oO?). `#` is a unary operator on a macro (it converts a macro to a string) so it should be handled by uncrustify as such and not insert a space (like we do with other unary operators like `++i`, `!cond`, etc.